tools/moonbeam-xcmp.ts

```diff
@@ -79,53 +84,58 @@ async function test() {
     "wrong balance for relayAlice, expected: 1000000000000000000, returned: " +
       Number(relayAlice.data.free)
   );
-  console.log("Sanity Checks Passed for Relay Chain and Both Parachains");
+  console.log("++ Sanity Checks Passed for Relay Chain and Both Parachains");
   // Open channel using relay sudo as caller
   const keyring = new Keyring({ type: "sr25519" });
   const alice = keyring.addFromUri("//Alice");
   const sender: number = 200;
   const recipient: number = 201;
-  const unsub = await relayApi.tx.sudo
-    .sudo(relayApi.tx.parasSudoWrapper.sudoEstablishHrmpChannel(sender, recipient, 8, 1024))
-    .signAndSend(alice, {}, (result) => {
-      console.log(`Current status is ${result.status}`);
-      if (result.status.isInBlock) {
-        console.log(`Transaction included at blockHash ${result.status.asInBlock}`);
-      } else if (result.status.isFinalized) {
-        console.log(`Transaction finalized at blockHash ${result.status.asFinalized}`);
-        unsub();
-      }
-    });
+  await new Promise<void>(async (resolve) => {
```

Comment:
@4meta5 the promise here garabties that the thread only continues after the tx is finalized